### PR TITLE
Update buildkit take2

### PR DESCRIPTION
**Pull request by thaJeztah**
quick try to make https://github.com/docker/cli/pull/3232 work

### CODE REVIEW

**Comment by thaJeztah** [comment]
Arf.. looks like this is causing issues; https://github.com/google/gnostic/commit/896953e6749863beec38e27029c804e88c3144b8.

```
error: unable to create file OpenAPIv2/OpenAPIv2.pb.go: File exists
error: unable to create file OpenAPIv2/OpenAPIv2.proto: File exists
error: unable to create file OpenAPIv2/README.md: File exists
error: unable to create file OpenAPIv2/openapi-2.0.json: File exists
error: unable to create file OpenAPIv3/OpenAPIv3.go: File exists
error: unable to create file OpenAPIv3/OpenAPIv3.pb.go: File exists
error: unable to create file OpenAPIv3/OpenAPIv3.proto: File exists
error: unable to create file OpenAPIv3/README.md: File exists
error: unable to create file OpenAPIv3/openapi-3.0.json: File exists
error: unable to create file OpenAPIv3/schema-generator/3.0.0.md: File exists
error: unable to create file OpenAPIv3/schema-generator/3.0.1.md: File exists
error: unable to create file OpenAPIv3/schema-generator/README.md: File exists
error: unable to create file OpenAPIv3/schema-generator/main.go: File exists
fatal: Could not reset index file to revision '7c663266750e7d82587642f65e60bc4083f1f84e'.
```

**Comment by thaJeztah** [comment]
And more errors currently;

```
/go/src/github.com/docker/cli/vendor/go.opentelemetry.io/proto/otlp/trace/v1/trace.pb.go:25:6: could not import go.opentelemetry.io/proto/otlp/common/v1 
/go/src/github.com/docker/cli/vendor/go.opentelemetry.io/proto/otlp/common/v1/common.pb.go:40:17: ProtoPackageIsVersion4 not declared by package proto)
```

**Comment by thaJeztah** [comment]
oh!

```
/go/src/github.com/docker/cli/cli/command/image/build_buildkit.go:403:9: v declared but not used 
```

File: vendor.conf [hunk 1]
```diff
@@ -67,9 +64,13 @@ github.com/theupdateframework/notary                bf96a202a09a312ae005cd312fc0
 github.com/tonistiigi/fsutil                        d72af97c0eaf93c1d20360e3cb9c63c223675b83
 github.com/tonistiigi/go-rosetta                    f79598599c5d34ea253b56a1d7c89bc6a96de7db
 github.com/tonistiigi/units                         6950e57a87eaf136bbe44ef2ec8e75b9e3569de2
+github.com/tonistiigi/vt100                         8066bb97264f56c603341d3fce837bb504e662ef # v0.0.0-20210615222946-8066bb97264f
 github.com/xeipuuv/gojsonpointer                    02993c407bfbf5f6dae44c4f4b1cf6a39b5fc5bb
 github.com/xeipuuv/gojsonreference                  bd5ef7bd5415a7ac448318e64f11a24cd21e594b
 github.com/xeipuuv/gojsonschema                     82fcdeb203eb6ab2a67d0a623d9c19e5e5a64927 # v1.2.0
+go.opentelemetry.io/contrib                         91e5180ae03828601f705ac8e7fc3bd9e34dae6a # v0.21.0
```
Comment (thaJeztah) [inline vendor.conf:71]: We need to check if we need the fork for this; https://github.com/moby/buildkit/blob/v0.9.0/go.mod#L79-L83

Comment (crazy-max) [reply vendor.conf:71]: Yes indeed I didn't take these ones into account. I think it was temporary and should be fixed upstream soon hopefully:
* open-telemetry/opentelemetry-go-contrib#873
* open-telemetry/opentelemetry-go-contrib#874
* open-telemetry/opentelemetry-go-contrib#875

Comment (thaJeztah) [reply vendor.conf]: Added a commit to use that fork

**Pull request closed without merging**